### ISSUE TITLE
Broken commit in master

### ISSUE THREAD

**Issue by ssiccha** [gap-packages/hap#46]
The commit 7f676756a5 touches 101 files. I guess that was an accident and in that case it should be reverted.

**Comment by ssiccha** [comment]
@grahamknockillaree

**Comment by alex-konovalov** [comment]
Yes, looks like updating Travis (or, more precisely, migrating from Travis CI to GitHub actions) went wrong. First, it does not require update of all other files. Second, the model setup from https://github.com/gap-packages/example does not have `.travis.yml` any more. This file is for Travis CI and it should go. The right files are in https://github.com/gap-packages/example/tree/master/.github/workflows. The switch should look similar to e.g. https://github.com/gap-packages/wedderga/pull/86/ which only changes 3 files.

**Comment by alex-konovalov** [comment]
@grahamknockillaree and actually @fingolfin did this in #47 - so one could revert commit 7f67675, and then merge #47. Can we do this for you?

**Comment by grahamknockillaree** [comment]
@alex-konovalov  >Can we do this for you?

YES PLEASE!   :)

**Comment by grahamknockillaree** [comment]
HAP now passes all tests, using either Polymake V3.2 or V4.3, provided Polymaking version 0.8.4 is present.

**Comment by fingolfin** [comment]
Great!

**Comment by fingolfin** [comment]
However, it seems you did not yet push this? The last commit here was 2 days ago and failed the CI tests

**Comment by grahamknockillaree** [comment]
I added a couple of lines to two files in Polymaking and updated the version of Polymaking to V0.8.4 . However, I don't think I have permission to release a new version of Polymaking. Once Polymaking V.0.8.4 is release then I think all tests should pass. (At least, they pass on my laptop.

**Comment by fingolfin** [comment]
You do have permissions to make a release of polymaking...

**Comment by grahamknockillaree** [comment]
Thanks. Then I'll try again!

**Comment by grahamknockillaree** [comment]
It seems that I did succeed in releasing V0.8.4 of Polymaking.  However, when testing HAP, GitHub is loading Polymaking V0.8.2 .  I thought that it should automatically use the latest release of Polymaking!? Not sure how to solve this.

**Comment by fingolfin** [comment]
No, the new package release also must be picked up by the GAP package distribution.

however, there is a way to install the latest version (and it doesn't even have to be a release!), which was part of my recent PR #48 which was unfortunately not merged. But you can copy it from there

**Comment by grahamknockillaree** [comment]
Thanks to lot's of help from @fingolfin  and @alex-konovalov the HAP tests now pass. (Well, one test using ListParallel failed. But that is a very quirky use of GAP, involving ssh calls to parallel GAP sessions. So I've simply deleted that test for the moment.)  In CI.yml I've added two instances of

GAP_PKGS_TO_CLONE: "polymaking hapcryst" # We need >= 0.8.4 and 0.1.13 to support polymake 4.0
